pkg/autodetect/operatorconfig.go

```diff
@@ -39,6 +39,21 @@ func (p ESOperatorIntegration) String() string {
 	return [...]string{"Yes", "No"}[p]
 }
 
+// KafkaOperatorIntegration holds the if the Kafka Operator integration is enabled.
+type KafkaOperatorIntegration int
+
+const (
+	// KafkaOperatorIntegrationYes represents the Kafka Operator integration is enabled.
+	KafkaOperatorIntegrationYes KafkaOperatorIntegration = iota
```

Comment:
Does it make sense to start with `KafkaOperatorIntegrationUnknown`? Otherwise default is yes?

Reply:
Default is no